chain/network/src/network_protocol/network.proto

```diff
@@ -189,19 +189,32 @@ message PeerAddr {
 }
 
 message AccountData {
-  string account_id = 1; // required
-  // Either address of the node handling the account (if it has a public IP),
-  // or a list of peers known to be connected to the validator.
+  reserved 1,3;
+
+  // PeerId of the node owning the account_key.
+  // Used to route the message over TIER1.
+  // TODO(gprusak): it should be possible to add support for routing
+  // messages to an account_id directly (for TIER1), instead of routing
+  // to a specific peer_id. Then this field won't be necessary.
+  // Unless we use it instead of AnnounceAccount.
+  PublicKey peer_id = 5; // required.
+ 
+  PublicKey account_key = 6; // required.
+
+  // List of nodes which
+  // - are trusted by the validator and
+  // - are connected to the validator directly
+  // - are willing to proxy traffic to the validator.
+  // It may include the validator node itself, if it has a public IP.
   // If empty, the validator explicitly declares that it has no public IP
-  // and the P2P routing should be used instead (discouraged, might be disallowed in the future).
-  repeated PeerAddr peers = 2;
-
-  // Epoch for which this data is valid.
-  // This AccountData should be signed with the account key assigned to
-  // <account_id> for epoch <epoch_id>.
-  CryptoHash epoch_id = 3;
-  // If there are multiple signed AccountData messages for the same
-  // account_id for the same epoch, the one with the most recent timestamp is valid.
+  // and the TIER2 routing should be used instead (discouraged, might be disallowed in the future).
+  repeated PeerAddr proxies = 2;
+
+  // Version of the AccountData. A node can override a previous version,
+  // by broadcasting a never version.
+  uint64 version = 7;
+  // Time of creation of this AccountData.
+  // TODO(gprusak): consider expiring the AccountData based on this field.
   google.protobuf.Timestamp timestamp = 4; 
```

Comment:
timestamp is the UTC time at which the AccountData was created. There is no guarantee that UTC clock will be monotone, and we want to be able to override the previous version of AccountData even if UTC clock moved backwards in time.

Reply:
So we can do smth like `min(prev_timestamp + 1, utc::now())`, basically force monotonicity

Reply:
ok, look, I want to actually track the time at which the thing was signed, so that I can view it and debug.
And I don't want a situation in which somebody at some point produces a piece of data 100 years into the future and then suddenly the timestamp is broken for the next 100 years.

I also want to eventually implement expiration policy on these data and given that the clocks on different machines are skewed it will really make it hard to reason about this if we add custom behavior of max(prev_timestamp+1,now()) on top of that.